src/platforms/windows/daemon/windowssplittunnel.cpp

```diff
@@ -493,7 +493,7 @@ QString WindowsSplitTunnel::convertPath(const QString& path) {
     // device should contain : for e.g C:
     return "";
   }
-  QByteArray buffer(2048, 0xFF);
```

Comment:
Given we now compile with warnings as errors, my MSVC compiler complained about the static type casting.